products/podman.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Podman Engine
+category: app
+iconSlug: podman
+permalink: /podman-engine
+alternate_urls:
+-   /podman
+versionCommand: podman --remote version --format '{{.Server.Version}}'
+changelogTemplate: "https://github.com/containers/podman/releases/tag/v__LATEST__"
+releaseDateColumn: true
+
+identifiers:
+-   repology: podman
+
+auto:
+  methods:
+  -   git: https://github.com/containers/podman.git
+
+releases:
+-   releaseCycle: "5.2"
+    releaseDate: 2024-08-02
+    eol: false
+    latest: "5.2.0"
+    latestReleaseDate: 2024-08-02
```

Comment:
```suggestion
-   releaseCycle: "5.2"
    releaseDate: 2024-08-01
    eol: false
    latest: "5.2.0"
    latestReleaseDate: 2024-08-01
```

See https://github.com/containers/podman/releases/tag/v5.2.0 (our automation scripts are running using UTC).

Reply:
There were some other document saying that it was localtime, but should be only to tweak the git script

Reply:
It now says `    "published_at": "2024-08-02T12:04:56Z",` so either change, or was looking at created_at?